nixos/modules/services/development/livebook.nix

```diff
@@ -0,0 +1,101 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.livebook;
+in
+{
+  options.services.livebook = {
+    enableUserService = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Whether to enable a user service for Livebook.  This implies
+        `installUserService`.
+      '';
+    };
+
+    installUserService = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Whether to install a user service for Livebook.  Unless
+        `enableUserService` is also set, users will have to manually activate
+        the service with `systemctl --user enable livebook`.
+      '';
+    };
```

Comment:
I don't see how this option is useful. If someone wants to generate the service but disable it temporarily for testing purposes, they could set `systemd.user.services.livebook.enable = lib.mkForce false;`. Unless there's a good reason why think lots of people would use it, I think we should drop it.

Reply:
To be honest, I don't see why it's useful either, but a bunch of the other user services have it, so I added it too.  For example `services.emacs.install`.  I'm happy to remove it.  Should I?

Reply:
Alright.  Let's go for simplicity, especially since people can override anyway with `mkForce`.